src/test/java/hw1/DataProvidersTest.java

```diff
@@ -0,0 +1,48 @@
+package hw1;
+
+import org.testng.annotations.DataProvider;
+
+public class DataProvidersTest {
+    @DataProvider
+    public static Object[][] doubleNumbers(){
+        return new Object[][]{
+                {7.01, -17.8},
+                {0.01, 0.99},
+                {128768.1, 9812.2},
+                {-98.2, -98.2},
+                {0, 76.9},
+                {0, 0},
+                {6.99, 0},
+                {-3.4, 3.4},
+                {Double.MIN_VALUE, Double.MAX_VALUE}
+        };
+    }
```

Comment:
Data providers that provide only input values to tests, which then compare the result with an etalon solution (method) used sometimes. But in practice, providers that provide expected result, as well as input values are used more widely. This can be critical in terms of resources, time.

So I would ask you to add expected results to data providers for this homework.

Reply:
done